### ISSUE TITLE
Following Feed

### ISSUE THREAD

**Issue by celozaga** [stevenlafl/threads-web-client#19]
The following feed was implemented in the latest iOS and Android update, will it also be implemented in this client?

**Comment by stevenlafl** [comment]
Yeah, there's some missing things I'll look at adding. Right now I'm busy with travel but that's on my list. I put a version number into the right sidebar, so when that gets updated it will turn red to notify you of an update.
